### DISCUSSION TITLE
Cannot find Y value in Entry from CartesianLayerModel

### DISCUSSION THREAD

**Discussion by matpascaud** [patrykandpatrick/vico#569] [Questions]
Hi,
In version 2.0.0, Im unable to get the Y value in order to add a markerComponent. 
`val markerComponent = MarkerComponent(txtComponent, null, null)
        markerComponent.labelFormatter = MarkerLabelFormatter { markedEntries, chartValues ->
            String.format("%.0f", markedEntries.first().entry.y) //only x available
        }`
        
 Did I miss something compare to the previous version?

**Comment by patrickmichalik** [comment]
Hello! Unlike `ChartEntry`, `CartesianLayerModel.Entry` doesn’t have a `y` field. This is because an entry doesn’t necessarily have to have a single _y_ value (or even a single main _y_ value). For example, in candlestick charts, which will be introduced in future Vico 2 alphas, one entry has four _y_ values. `ColumnCartesianLayerModel.Entry` and `LineCartesianLayerModel.Entry` do have `y` fields, however. If you have just one layer type, it’s sufficient to use casting. Otherwise, a solution such as [this one](https://github.com/patrykandpatrick/vico/blob/v2.0.0-alpha.6/vico/core/src/main/java/com/patrykandpatrick/vico/core/marker/DefaultMarkerLabelFormatter.kt#L56-L62) can be utilized.
